go/cmd/vtgateclienttest/services/success.go

```diff
@@ -32,11 +32,11 @@ func (c *successClient) Begin(ctx context.Context) (*vtgatepb.Session, error) {
 	}, nil
 }
 
-func (c *successClient) Commit(ctx context.Context, session *vtgatepb.Session) error {
+func (c *successClient) Commit(ctx context.Context, twopc bool, session *vtgatepb.Session) error {
```

Comment:
Yes, one code path should return success here. I think you'd do it by having the dtid start with success... and then in that case you return success. Then the client passes the right dtid, and checks it worked. But you need the client side to be implemented for this, so maybe wait until you work on that part...